vme/src/dilrun.cpp

```diff
@@ -271,8 +311,22 @@ int dil_intr_insert(dilprg *p, ubit8 *lab, ubit8 *elab, ubit16 flags)
 
     return intnum;
 }
+
+
 void dil_free_var(dilvar *v)
 {
+    if (v->name)
+    {
+        FREE(v->name);
+        v->name = nullptr;
+    }
+
+    if (v->itype == DilIType_e::Global)
+    {
+        // It's a global variable, don't free it.
```

Comment:
Is it ok for it not to have a name then - since that gets free'ed above

Reply:
Its ok but confusing, so I swapped it around. Thanks.